controllers/rollingupgrade_controller.go

```diff
@@ -875,3 +908,35 @@ func (r *RollingUpgradeReconciler) UpdateInstance(ctx *context.Context,
 	ch <- nil
 	return
 }
+
+func requiresRefresh(ec2Instance *autoscaling.Instance, definition *launchDefinition) bool {
+	if definition.kind == launchKindLaunchConfiguration {
+		if ec2Instance.LaunchConfigurationName != nil && *ec2Instance.LaunchConfigurationName != *definition.launchConfigurationName {
```

Comment:
Are we comparing pointers in `*ec2Instance.LaunchConfigurationName != *definition.launchConfigurationName`?
I think this line can be `if aws.StringValue(ec2Instance.LaunchConfigurationName) != nil { ... }`

Reply:
good catch.

Reply:
Fixed.